### PR TITLE
[v7r1] fixes for MultiProcessor queues

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*Configuration
CHANGE: Adding Pool as LocalCEType for MultiProcessor queues (invoked by BDII2CSAgent)

*WMS
CHANGE: JobAgent: getting CEType only from LocalSite/LocalCE config
CHANGE: JobAgent: flipped the default for fillingMode
CHANGE: SiteDirector: The max number of jobs in filling mode is not set anymore by the SD. The option MaxJobsInFillMode  was removed

*Core
FIX: ElasticSearchDB: fix exception handling when checking existence of indices

ENDRELEASENOTES

### CODE REVIEW

File: WorkloadManagementSystem/Agent/SiteDirector.py [hunk 1]
```diff
@@ -1149,24 +1145,14 @@ def _getPilotOptions(self, queue, pilotsToSubmit, **kwargs):
     ownerDN = self.pilotDN
     ownerGroup = self.pilotGroup
     # Request token for maximum pilot efficiency
-    result = gProxyManager.requestToken(ownerDN, ownerGroup, pilotsToSubmit * self.maxJobsInFillMode)
+    numberOfProcessors = queueDict.get('NumberOfProcessors', 1)
```
Comment (andresailer) [inline WorkloadManagementSystem/Agent/SiteDirector.py:1148]: Does this need to be cast to int?

File: docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst [hunk 2]
```diff
@@ -50,10 +50,11 @@ This sub-subsection specify the attributes of each particular CE of the site. Mu
...
 | *<CE_NAME>/LocalCEType*                        | Type of 'Inner' CE, normally empty. Default = "InProcess".   | LocalCEType = Pool             |
-|                                                | Possibilities: potentially all CEs type, but in practice     |                                |
+|                                                | Possibilities: potentially all CE types, but in practice     |                                |
 |                                                | the most valid would be: InProcess, Sudo, Singularity, Pool. |                                |
 |                                                | Pool CE in turn uses InProcess (Default)                     |                                |
 |                                                | or Sudo or Singularity. To specify, use Pool/ce_type.        | LocalCEType = Pool/Singularity |
+|                                                | This option can also go at the Queue level.                  |                                |
 +------------------------------------------------+------------------------------------------------------------- +--------------------------------+
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst:58]: There is a problem with the table syntax, it is missing a `-` in many of the `----` lines.

Comment (fstagni) [reply docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst:58]: I have fixed it here.

Comment (andresailer) [reply docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst:58]: You still have to push this fix, right?

File: docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst [hunk 3]
```diff
@@ -38,65 +38,68 @@ The three strings are concatenated with "." to produce the name of the sites.
 CEs  sub-subsection
 -------------------
 
-This sub-subsection specify the attributes of each particular CE of the site. Must be noticed than in each DIRAC site can be more than one CE.
+This sub-subsection specify the attributes of each particular CE of the site. Must be noticed than in eachDRAC site can be more than one CE.
```
Comment (andresailer) [inline docs/source/AdministratorGuide/Configuration/ConfReference/Resources/Sites/index.rst:41]:
```suggestion
This sub-subsection specifies the attributes of each particular CE of the site. For each DIRAC site there can be more than one CE.
```

**Review by atsareg — APPROVED** [review]

**Pull request merged**